proto/dwtools/abase/l0.test/Buffer.test.s

```diff
@@ -727,6 +742,11 @@ function constructorIsBuffer( test )
   var expected = false;
   test.identical( got, expected );
 
+  test.case = 'check BufferView from raw';
+  var got = _.constructorIsBuffer( new BufferView( new BufferRaw( 3 ) ) );
+  var expected = false;
+  test.identical( got, expected );
+
```

Comment:
All test routines is good.

Please, sort test routines. The order of test routines should be identical to order of main routines ( test units ).